src/app/components/forms/input/file-input/terra-file-input.component.spec.ts

```diff
@@ -99,8 +113,111 @@ describe('TerraFileInputComponent', () =>
         expect(component).toBeTruthy();
     });
 
-    afterAll(() =>
+    it('should accept TerraRegex.MIXED by default', () =>
+    {
+        expect(component.regex).toBe(TerraRegex.MIXED);
+    });
+
+    it('should have showPreview false by default', () =>
+    {
+        expect(component.inputShowPreview).toBeFalsy();
+    });
+
+    it('should have inputAllowFolders true by default', () =>
+    {
+        expect(component.inputAllowFolders).toBeTruthy();
+    });
+
+    it('should be an webImage if the extension is .jpg', () =>
+    {
+        expect(component.isWebImage(jpgFileName)).toBeTruthy();
+    });
+
+    it('should have publicUrl as value after selection', () =>
+    {
+        component.onObjectSelected(new TerraStorageObject(fileData.objects[0]));
+
+        expect(component.value).toBe(fileData.objects[0].publicUrl);
+    });
+
+    it('should have a icon class `jpg` if the file has a jpg extension', () =>
+    {
+        expect(component.getIconClass(jpgFileName)).toBe('icon-file_extension_jpg');
+    });
+
+    it('should have a icon class `jpg` if the file has a jpg extension', () =>
```

Comment:
ist der gleiche Testname wie oben
```suggestion
    it('should have a icon class `folder` if the file is a folder', () =>
```